fabric-content-registries-v0/src/main/java/net/fabricmc/fabric/api/registry/TillableBlockRegistry.java

```diff
@@ -81,6 +81,6 @@ public static void register(Block input, Predicate<ItemUsageContext> usagePredic
 	public static void register(Block input, Predicate<ItemUsageContext> usagePredicate, BlockState tilled, ItemConvertible droppedItem) {
 		Objects.requireNonNull(tilled, "tilled block state cannot be null");
 		Objects.requireNonNull(droppedItem, "dropped item cannot be null");
-		register(input, usagePredicate, HoeItem.getTillingConsumer(tilled, droppedItem));
+		register(input, usagePredicate, HoeItem.createTillAndDropAction(tilled, droppedItem));
```

Comment:
The javadoc of all overloads also references the old HoeItem method names so they have to be renamed as well.